src/main/java/org/broadinstitute/hellbender/tools/walkers/mutect/Mutect2FilteringEngine.java

```diff
@@ -73,14 +73,12 @@ public void applyGermlineVariantFilter(final VariantContext vc, final Collection
         final int loc = vc.getStart();
         final boolean siteInCosmic = !featureContext.getValues(MTAC.cosmicFeatureInput, loc).isEmpty();
         final boolean siteInDbsnp = !featureContext.getValues(MTAC.dbsnp.dbsnp, loc).isEmpty();
-        final double normalLodFilterThreshold = siteInDbsnp && !siteInCosmic ? MTAC.NORMAL_DBSNP_LOD_THRESHOLD : MTAC.NORMAL_LOD_THRESHOLD;
-
-        final double normalLod = vc.getAttributeAsDouble(GATKVCFConstants.NORMAL_LOD_KEY, 0.0);
-
-        if (normalLod < normalLodFilterThreshold) {
-            filters.add(GATKVCFConstants.GERMLINE_RISK_FILTER_NAME);
+        if (siteInDbsnp && !siteInCosmic ) {
```

Comment:
We already check the normal lod as part of the first pass.  This filter checks for dbSNP and then applies a *stricter* lod threshold.  Note that this reproduces old behavior (I think) and I'm only keeping it because I know we're about to change things.